typescript/src/ethereum.ts

```diff
@@ -57,8 +57,11 @@ export class Bridge implements ChainBridge {
     // Build the redemption key by using the 0x-prefixed wallet PKH and
     // prefixed output script.
     const redemptionKey = utils.solidityKeccak256(
-      ["bytes20", "bytes"],
-      [`0x${walletPubKeyHash}`, prefixedRawRedeemerOutputScript]
+      ["bytes32", "bytes20"],
```

Comment:
Can we also do that change in the mock implementation of the `Bridge`? This is located in `test/utils/mock-bridge.ts` file.

Reply:
0a6746a6ede9534dbd6fc67b8ffc18dd4791c0ab